react/utils/index.js

```diff
@@ -21,10 +21,14 @@ export function setStyle(target, styles) {
   })
 }
 
+function _getZeroOrString(num = 0) {
+  return num === 0 ? num : `${num}%`
+}
+
 export function setTransformProperty(target, x = 0, y = 0, z = 0) {
```

Comment:
I know this function was not created in this PR, but this is not a good name—it sets only the translate property, not a general `transform` (which could be scaling or rotating too). consider renaming it, if it's not used too often